generator/osm2type.cpp

```diff
@@ -586,15 +586,12 @@ string DetermineSurface(OsmElement * p)
       (void)strings::to_double(tag.m_value, surfaceGrade);
     else if (tag.m_key == "tracktype")
       trackGrade = tag.m_value;
-    else if (tag.m_key == "highway")
+    else if (tag.m_key == "highway" && tag.m_value != "ford")
       highway = tag.m_value;
     else if (tag.m_key == "4wd_only" && (tag.m_value == "yes" || tag.m_value == "recommended"))
       return "unpaved_bad";
   }
 
-  if (highway.empty() || (surface.empty() && smoothness.empty()))
-    return {};
-
   // According to https://wiki.openstreetmap.org/wiki/Key:surface
   static base::StringIL pavedSurfaces = {
       "asphalt",  "cobblestone", "chipseal", "concrete",
```

Comment:
`concrete:plates` seems missing (or is it converted?)

Reply:
It'll be considered paved good, though the matching mechanic is not obvious, I'll add a comment.